documentation/guides/migrating-icons-from-v3-to-v4.md

```diff
@@ -0,0 +1,64 @@
+# Migrating from v3 to v4
```

Comment:
Maybe we should revised the title to make it more Icon obvious (ex: "# Migrating Polaris Icons from v3 to v4")